src/sql/workbench/contrib/notebooksExplorer/browser/notebookExplorerViewlet.ts

```diff
@@ -0,0 +1,149 @@
+	) {
+		super(VIEWLET_ID, instantiationService.createInstance(NotebookExplorerViewPaneContainer), telemetryService, storageService, instantiationService, themeService, contextMenuService, extensionService, contextService, layoutService, configurationService);
+	}
+}
+
+export class NotebookExplorerViewPaneContainer extends ViewPaneContainer {
+	private root: HTMLElement;
+
+	private notebookSourcesBox: HTMLElement;
+
+	constructor(
+		@IWorkbenchLayoutService layoutService: IWorkbenchLayoutService,
+		@ITelemetryService telemetryService: ITelemetryService,
+		@IInstantiationService instantiationService: IInstantiationService,
+		@IThemeService themeService: IThemeService,
+		@IStorageService storageService: IStorageService,
+		@IWorkspaceContextService contextService: IWorkspaceContextService,
+		@IContextMenuService contextMenuService: IContextMenuService,
+		@IExtensionService extensionService: IExtensionService,
+		@IConfigurationService configurationService: IConfigurationService,
+		@IMenuService private menuService: IMenuService,
+		@IContextKeyService private contextKeyService: IContextKeyService,
+		@IViewDescriptorService viewDescriptorService: IViewDescriptorService
+	) {
+		super(VIEWLET_ID, { mergeViewWithContainerWhenSingleView: true }, instantiationService, configurationService, layoutService, contextMenuService, telemetryService, extensionService, themeService, storageService, contextService, viewDescriptorService);
+	}
+
+	create(parent: HTMLElement): void {
+		addClass(parent, 'notebookExplorer-viewlet');
+		this.root = parent;
+
+		this.notebookSourcesBox = append(this.root, $('.notebookSources'));
+
+		return super.create(this.notebookSourcesBox);
+	}
+
+	public updateStyles(): void {
+		super.updateStyles();
+	}
+
+	focus(): void {
+	}
+
+	layout(dimension: Dimension): void {
+		toggleClass(this.root, 'narrow', dimension.width <= 300);
+		super.layout(new Dimension(dimension.width, dimension.height));
+	}
+
+	getOptimalWidth(): number {
+		return 400;
+	}
+
+	getSecondaryActions(): IAction[] {
+		let menu = this.menuService.createMenu(MenuId.NotebookExplorerAction, this.contextKeyService);
+		let actions = [];
+		menu.getActions({}).forEach(group => {
+			if (group[0] === 'secondary') {
+				actions.push(...group[1]);
+			}
+		});
+		menu.dispose();
+		return actions;
+	}
+
+	protected createView(viewDescriptor: IViewDescriptor, options: IViewletViewOptions): ViewPane {
+		let viewletPanel = this.instantiationService.createInstance(viewDescriptor.ctorDescriptor.ctor, options) as ViewPane;
+		this._register(viewletPanel);
+		return viewletPanel;
+	}
+}
+
+export const NOTEBOOK_VIEW_CONTAINER = Registry.as<IViewContainersRegistry>(ViewContainerExtensions.ViewContainersRegistry).registerViewContainer({
+	id: VIEWLET_ID,
+	name: localize('notebookExplorer.name', "Notebooks"),
+	ctorDescriptor: new SyncDescriptor(NotebookExplorerViewPaneContainer),
+	icon: 'notebookExplorer',
+	order: 6,
```

Comment:
@anthonydresser  I've given the order 6 but it still shows up at the top above connections viewlet. Can you help me  on how to fix this order thing.